Docker.DotNet.X509/CertificateCredentials.cs

```diff
@@ -18,6 +19,19 @@ public CertificateCredentials(X509Certificate2 clientCertificate)
             _certificate = clientCertificate;
         }
 
+#if (NET45 || NET46)
+        public CertificateCredentials(string pfxFilePath, SecureString password)
```

Comment:
This approach works. Do we think its better to do it this way or to have a constructor:

```C#
public CertificateCredentials(X509Certificate2)
{
}
```

And then

```C#
public static X509Certificate2 GetCertificateFromPFX(str, pass);
public static X509Certificate2 GetCertificateFromPEM(str, str);
```

I tend to like this approach because it means we can add an 'unlimited' number of 'named' helpers to get an X509Cert but dont need to have constructors that might conflict in some ways.